src/address.js

```diff
@@ -181,6 +183,28 @@ export function addOrReplaceAddressTypeOnList(addresses, newAddress) {
   return newAddresses
 }
 
+export function addOrReplaceDeliveryAddressOnList(addresses, newAddress) {
```

Comment:
```suggestion
export function saveDeliveryAddressOnList(addresses, newAddress) {
```

Reply:
That wouldnt be the Pattern on this file If you see the other functions.
Set is also a verb to express this better If we would change it